install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml

```diff
@@ -76,7 +76,7 @@ spec:
               command:
                 - /usr/local/bin/sidecar-injector
                 - probe
-                - --probe-path=/health
+                - --probe-path=/tmp/health
```

Comment:
Same here with `emptyDir`

Reply:
@johscheuer I think we should have another PR to support galley to run with `readOnlyRootFilesystem` instead of adding it in this PR. let us do it after this PR is in. Thanks.

Reply:
Yeah, that's fine.

Reply:
Thanks